lib/post/utils/comment_action_helpers.dart

```diff
@@ -55,6 +58,11 @@ final List<ExtendedCommentCardActions> commentCardDefaultActionItems = [
     icon: Icons.share_rounded,
     label: AppLocalizations.of(GlobalContext.context)!.shareLink,
   ),
+  ExtendedCommentCardActions(
+    commentCardAction: CommentCardAction.report,
+    icon: Icons.report_outlined,
+    label: '${AppLocalizations.of(GlobalContext.context)!.report} ${AppLocalizations.of(GlobalContext.context)!.comment}',
```

Comment:
Just curious about the use of concatenating resource strings like this. It works in English, but maybe in other languages with different word order, it might not make sense to translate them separately and then combine. I'd recommend a single resource for this.

Reply:
Resolved.